Welcome cleanup (#581)
dfinity/docs: modules/introduction/pages/welcome.adoc

If you’ve landed here, you’re interested in learning more about the Internet Computer. You’re in the right place — take a look below for where to get started!

> NOTE: Qualified developers can access $20 worth of free cycles to begin deploying canister smart contracts to the Internet Computer. https://faucet.dfinity.org/auth[Claim your free cycles]

Author: ```suggestion
NOTE: Qualified developers can access $20 worth of free cycles to begin deploying canister smart contracts to the Internet Computer blockchain. https://faucet.dfinity.org/auth[Claim your free cycles]
```

Interested in learning more? Check out the following resources:

* link:https://www.youtube.com/watch?v=YWHTNr8RZHg&list=PLuhDt1vhGcrf4DgKZecU3ar_RA1cB0vUT&index=17&ab_channel=DFINITY[Primer to the Internet Computer], a high-level overview of the Internet Computer in under 10 minutes
* link:https://medium.com/dfinity[The Internet Computer Review], our blog covering updates for the Internet Computer 
* link:https://thereboot.com/[The Reboot], our tech publication exploring issues with the current internet

Follow us on the official link:https://twitter.com/dfinity[DFINITY Twitter] for the latest updates.

[[for-developers]]
== For: Developers
The Internet Computer hosts special smart contracts, called link:https://medium.com/dfinity/software-canisters-an-evolution-of-smart-contracts-internet-computer-f1f92f1bfffb[canisters]. A canister smart contract is a bundle of link:https://webassembly.org/[WebAssembly] bytecode logic and memory pages. Dapps on the Internet Computer are created from one or more canisters. 

> Canister smart contracts can serve web content directly to end-users, and users can interact with blockchain services without holding tokens. When building on the Internet Computer, you can deploy code directly to the Internet. 

Author: ```suggestion
Canister smart contracts can serve web content directly to end-users, e.g., in their browsers, and users can interact with blockchain services without holding tokens. 
```

> The Internet Computer can eventually host any number of canister smart contractss (i.e., code & state). Moreover, it can run them concurrently with unbounded on-chain capacity. This means you can create dapps that scale. 

Author: ```suggestion
The Internet Computer blockchain can eventually host any number of canister smart contracts (i.e., code & state). Moreover, it can run them concurrently with unbounded on-chain capacity. This means you can create dapps that scale. 
```

[[for-protocol-enthusiasts]]
== For: Protocol Enthusiasts

> The Internet Computer is created by the Internet Computer Protocol (“ICP”), which has formed the world’s first web-speed, web-serving public blockchain. The Internet Computer can grow its capacity with demand and is self-governing. It combines special node machines run en masse by independent data centers all around the world. Like all blockchains, it is unstoppable, and the code it hosts is tamperproof. 

Reviewer: Maybe "as required", instead of "with demand"
```suggestion
The Internet Computer is created by the Internet Computer Protocol (“ICP”), which has formed the world’s first web-speed, web-serving public blockchain. The Internet Computer is self-governing and can grow its capacity as required. It combines special node machines run en masse by independent data centers all around the world. Like all blockchains, it is unstoppable, and the code it hosts is tamperproof. 
```

Get started by checking out our “Concepts” section to the left, starting with xref:developers-guide:concepts/what-is-IC.adoc[What is the Internet Computer?]

To learn more about the Internet Computer Protocol, check out the following resources:

* link:https://dfinity.org/technicals/[Technical Library], for in-depth videos by our world-class R&D team explaining components of the Internet Computer Protocol. Worth noting in particular:
** link:https://dfinity.org/technicals/chain-key-technology[Chain Key Cryptography], one of the fundamental breakthroughs enabling the Internet Computer to scale to millions of nodes. The most notable innovation of Chain Key cryptography is that the Internet Computer has a single public key, which enables any device to verify the authenticity of artifacts generated by the Internet Computer, even smart watches and mobile phones.
* xref:interface-spec:index.adoc[Interface Specification], for a deeply technical document that provides an overview of the lower-level external interfaces of the Internet Computer

> * link:https://dashboard.internetcomputer.org/[Internet Computer Dashboard], to monitor real-time metrics around the Internet Computer

Author: ```suggestion
* link:https://dashboard.internetcomputer.org/[Internet Computer Dashboard], to monitor real-time metrics around the Internet Computer blockchain
```
